pkg/apis/externaldns/types.go

```diff
@@ -556,7 +558,7 @@ func App(cfg *Config) *kingpin.Application {
 	app.Flag("exoscale-apisecret", "Provide your API Secret for the Exoscale provider").Default(defaultConfig.ExoscaleAPISecret).StringVar(&cfg.ExoscaleAPISecret)
 
 	// Flags related to RFC2136 provider
-	app.Flag("rfc2136-host", "When using the RFC2136 provider, specify the host of the DNS server").Default(defaultConfig.RFC2136Host).StringVar(&cfg.RFC2136Host)
+	app.Flag("rfc2136-host", "When using the RFC2136 provider, specify the host of the DNS server (optionally specify multiple times when when using --rfc2136-load-balancing-strategy)").Default(defaultConfig.RFC2136Host[0]).StringsVar(&cfg.RFC2136Host)
```

Comment:
Why would we need default? Does it mean now it's always enabled?

For optional not values, not sure if default is recommended

Reply:
It was default before that a look, the only part that's optional is to specify multiple when using `--rfc2136-load-balancing-strategy` otherwise it uses the first in the list.